core/src/main/java/gyro/core/reference/FinderReferenceResolver.java

```diff
@@ -67,4 +72,19 @@ public Object resolve(Scope scope, List<Object> arguments) {
         return resources;
     }
 
+    private Map<String, Object> getFilteredFilters(Map<String, Object> argumentFilters, FinderType<? extends Finder<Resource>> finderType) {
```

Comment:
Is there a better name for this method than `getFilteredFilters`? It's not so much filtering as it is translating/mapping the Gyro field name to the provided `@Filter` name within the filter map.

Might be useful to provide a comment explaining what this is doing.